seed/seedwriter/writer.go

```diff
@@ -56,6 +56,15 @@ type Options struct {
 	ManifestPath string
 }
 
+// initManifest returns either the manifest already provided by the
+// options, or if not provided, returns a newly initialized manifest.
+func (opts *Options) initManifest() *Manifest {
```

Comment:
this could just be called manifest() here

Reply:
yes I was actually doubting this one, so appreciate your input here!